sdk/notificationhubs/notification-hubs/CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Release History
 
+## 1.0.0-beta.2 (2022-08-17)
+
+- Removed internal dependencies of `@azure/core-amqp` and inlined code
+- Added Telemetry using the `X-MS-AZSDK-Telemetry` header.
```

Comment:
nit: can we have all parts of this header name with the same casing?

Reply:
Fixing